tests/test_client_session.py

```diff
@@ -493,6 +493,26 @@ class UnexpectedException(BaseException):
     assert resp_cookies["response"].value == "resp_value"
 
 
+async def test_cookies_load_on_request(aiohttp_client) -> None:
+    async def handler(_):
+        return web.Response()
+
+    app = web.Application()
+    app.router.add_route('GET', '/', handler)
+
+    # no cookies should be set at init
+    session = await aiohttp_client(app, cookies=None)
+
+    with mock.patch("http.cookies.SimpleCookie.load") as patched:
```

Comment:
Drop mocks, please.
Making sure that `SimpleCookie.load()` is called means nothing from the library perspective.
It doesn't guarantee to keep the feature working after code refactoring.
The test purpose is not obvious.
Explicit check for sent cookies on the server side is much better.